bolt/include/bolt/Passes/InsertNegateRAStatePass.h

```diff
@@ -16,7 +27,15 @@ class InsertNegateRAState : public BinaryFunctionPass {
   /// Pass entry point
   Error runOnFunctions(BinaryContext &BC) override;
   void runOnFunction(BinaryFunction &BF);
+
+private:
+  /// Loops over all instructions and adds OpNegateRAState CFI
+  /// after any pointer signing or authenticating instructions.
+  /// Returns true, if any OpNegateRAState CFIs were added.
```

Comment:
nit: `\return` true if ..

Reply:
I don't see that notation anywhere in BOLT, so I'm not sure adding here improves much.

Reply:
This is more on [doxygen](https://llvm.org/docs/CodingStandards.html#doxygen-use-in-documentation-comments) guidelines. But I also see this in a few places, eg [findAttributeInfo](https://github.com/llvm/llvm-project/blob/6b1db79887df19bc8e8c946108966aa6021c8b87/bolt/include/bolt/Core/DebugData.h#L61).

This is a nit anyway, so if you remember add it on a follow-up change, when we have some update on tests or the scheduling policy, etc.